tests/test_unit.py

```diff
@@ -26,3 +35,49 @@ def test_interpreter(v1, v2, equal):
         assert Interpreter(v1) != Interpreter(v2)
         assert hash(Interpreter(v1)) != hash(Interpreter(v2))
         assert repr(Interpreter(v1)) != repr(Interpreter(v2))
+
+
+def test_venv_path(current_interpreter: Interpreter) -> None:
+    py_version = "".join((str(_) for _ in sys.version_info[:3]))
+    assert current_interpreter.venv_path() == ".riot/venv_py{}".format(py_version)
+
+
+def test_sitepackages_path(current_interpreter: Interpreter) -> None:
+    py_dot_version = ".".join((str(_) for _ in sys.version_info[:2]))
+
+    expected = os.path.join(
+        current_interpreter.venv_path(),
+        "lib/python{}/site-packages".format(py_dot_version),
+    )
+    assert current_interpreter.site_packages_path().endswith(expected)
```

Comment:
Might be worth using `os.path.join` to also concatenate the bits in `lib/python{}/site-packages`, and perhaps convert to an absolute path with `os.path.abs` and do exact matching, in case this will ever need to be run on other platforms, e.g. win?

Reply:
yeah, join works well, good suggestion